addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -453,6 +462,346 @@ QUnit.test('do not post message with "Enter" keyboard shortcut', async function
+        id: 10,
+        email_cc: "john@test.be",
+        partner_ids: [100],
+    });
+    await this.start();
+    const chatter = this.env.models['mail.chatter'].create({
+        threadId: 10,
+        threadModel: 'res.fake',
+    });
+    await this.createChatterComponent({ chatter });
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatterTopbar_buttonSendMessage`).click()
+    );
+    const title = document.querySelector('.o_ComposerSuggestedRecipient:not([data-partner-id])').getAttribute('title');
+    assert.strictEqual(
+        title,
+        "Add as recipient and follower (reason: CC email)",
+        "display reason for suggested recipient without partner on mouse over",
+    );
+    const partnerTitle = document.querySelector('.o_ComposerSuggestedRecipient[data-partner-id="100"]').getAttribute('title');
+    assert.strictEqual(
+        partnerTitle,
+        "Add as recipient and follower (reason: Email partner)",
+        "display reason for suggested recipient with partner on mouse over",
+    );
+});
+
+QUnit.test("Suggested recipient without partner unchecked by default", async function (assert) {
+    assert.expect(1);
+
+    this.data['res.fake'].records.push({
+        id: 10,
+        email_cc: "john@test.be",
+    });
+    await this.start();
+    const chatter = this.env.models['mail.chatter'].create({
+        threadId: 10,
+        threadModel: 'res.fake',
+    });
+    await this.createChatterComponent({ chatter });
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatterTopbar_buttonSendMessage`).click()
+    );
+    const checkboxUnchecked = document.querySelector('.o_ComposerSuggestedRecipient:not([data-partner-id]) input[type=checkbox]');
+    assert.notOk(
+        checkboxUnchecked.checked,
+        "suggested recipient without a partner should not be checked by default",
+    );
+});
+
+QUnit.test("Suggested recipient with partner checked by default", async function (assert) {
+    assert.expect(1);
+
+    this.data['res.partner'].records.push({
+        display_name: "John Jane",
+        email: "john@jane.be",
+        id: 100,
+    });
+    this.data['res.fake'].records.push({
+        id: 10,
+        partner_ids: [100],
+    });
+    await this.start();
+    const chatter = this.env.models['mail.chatter'].create({
+        threadId: 10,
+        threadModel: 'res.fake',
+    });
+    await this.createChatterComponent({ chatter });
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatterTopbar_buttonSendMessage`).click()
+    );
+    const checkboxChecked = document.querySelector('.o_ComposerSuggestedRecipient[data-partner-id="100"] input[type=checkbox]');
+    assert.ok(
+        checkboxChecked.checked,
+        "suggested recipient with a partner should be checked by default.",
+    );
+});
+
+QUnit.test("suggested recipients list display a \"show more\" button when >= 4 are available", async function (assert) {
```

Comment:
_"more than 3 suggested recipients: display only 3 + 'show more' button"_

Also avoid escaping quotes with `\` in test name and assertion name, it's annoying to grep... use single quotes around test name or even backquote.

Reply:
I used `only 3 and 'show more'` as someone could understand `3 +` as more than 3.